src/platform/datapath_winuser.c

```diff
@@ -2465,7 +2469,7 @@ CxPlatSocketAllocRxIoBlock(
     if (SocketProc->Parent->UseRio) {
         OwningPool = &DatapathProc->RioRecvPool;
     } else {
-        OwningPool = &DatapathProc->RecvDatagramPool;
+        OwningPool = (CXPLAT_POOL*)&DatapathProc->RecvDatagramPool;
```

Comment:
It might be better to name the CXPLAT_POOL field of CXPLAT_POOL_EX in C, too, and then be able to use &Pool.Foo instead of a typecast and assumption about the field layouts.